references/segmentation/transforms.py

```diff
@@ -75,9 +74,15 @@ def __call__(self, image, target):
         return image, target
 
 
-class ToTensor(object):
+class PILToTensor:
     def __call__(self, image, target):
         image = F.pil_to_tensor(image)
+        target = torch.as_tensor(np.array(target), dtype=torch.int64)
```

Comment:
What is the type of target passed to this method? Can we avoid using numpy here?

Reply:
Done. I haven't been able to test this yet as I am again getting error on aws cluster. I'll update here once I have completed the training run.

Reply:
Seems to be Image type. Without `np.array()` it throws this error: `TypeError: an integer is required (got type Image)`